beacon-chain/rpc/apimiddleware/custom_handlers.go

```diff
@@ -193,17 +201,16 @@ func handlePostSSZ(
 	return true
 }
 
-func sszRequested(req *http.Request) bool {
-	accept, ok := req.Header["Accept"]
-	if !ok {
-		return false
+func sszRequested(req *http.Request) (bool, error) {
+	acceptableTypes := []contenttype.MediaType{
```

Comment:
Could we accomplish this without a new dependency? It doesn't seem like this library is doing much heavy lifting for us.

Reply:
+1 , less dependencies the better.